receiver/hostmetricsreceiver/internal/scraper/loadscraper/config.go

```diff
@@ -19,4 +19,7 @@ import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostm
 // Config relating to Load Metric Scraper.
 type Config struct {
 	internal.ConfigSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
+
+	// If true, metrics will be average load per cpu
+	PerCPU bool `mapstructure:"per_cpu"`
```

Comment:
This is not really "per_cpu" (which for me means that is broken down by cpu core), is more or less average per cpu, correct?

Reply:
this is absolutely correct. I confess I simply copied the existing option from smart agent: https://docs.signalfx.com/en/latest/integrations/agent/monitors/load.html#configuration

do you want I change it for something like `average_per_cpu` ?